init-interfaces.sh

```diff
@@ -42,16 +42,18 @@ set_description() {
   local mac=$1
   local nic=$2
   local description=$3
```

Comment:
Please make the commit title `Works better with spaces in file name` clearer, and add commit body if needed.